module/zfs/zio.c

```diff
@@ -89,6 +89,11 @@ static uint_t zio_slow_io_ms = (30 * MILLISEC);
 #define	BP_SPANB(indblkshift, level) \
 	(((uint64_t)1) << ((level) * ((indblkshift) - SPA_BLKPTRSHIFT)))
 #define	COMPARE_META_LEVEL	0x80000000ul
+
+/* Get allocation size between spa_min_alloc and spa_gcd_alloc */
+#define SPA_ALLOC_SIZE(size, min_alloc, gcd_alloc) \
+	((size > min_alloc) ? gcd_alloc : min_alloc)
+
```

Comment:
I meant something like this:
uint64_t
spa_min_alloc_size(spa_t *spa, uint64_t size)
{
    if (size > spa->spa_min_alloc)
        return (roundup(size, spa->spa_gcd_alloc))
    return (spa->spa_min_alloc)
}